Dockerfile

```diff
@@ -19,7 +23,7 @@ RUN \
     cargo --version && \
     rustc --version && \
     mkdir -m 755 bin && \
-    cargo build --release && \
+    CONTILE_VERSION=${VERSION} cargo build --release && \
```

Comment:
FWIW, `$CONTILE_VERSION` is only used at compile time, so it's only referenced when we build Contile.